uint64 seq
uint64 width
uint64 height

# intrinsics, in pixels
float32 focal_x
float32 focal_y
float32 center_x
float32 center_y

# current sensor pose
geometry_msgs/Pose pose

# SYNTHETIC, generated by ElasticFusion
float32[] color    # 3 float per pixel
float32[] depth    # 1 float per pixel
float32[] position # 3 float per pixel

uint64[] guid         # 1 guid per pixel
uint32[] luid         # 1 luid per pixel
uint32[] luid_removed # luids of surfels removed in this frame
uint32 max_luid

float32[] normal  # 3 float per pixel
float32[] radius  # 1 float per pixel

# INPUT for this frame, from the RGB-D sensor
uint8[] input_color   # RGB, 3 byte per pixel
uint16[] input_depth  # DEPTH, in mm
